docs/pages/guides/new-architecture.mdx

```diff
@@ -0,0 +1,127 @@
+---
+title: New Architecture
+description: Learn about React Native's "New Architecture" and how and why to migrate to it
+---
+
+import { BookOpen02Icon } from '@expo/styleguide-icons';
+import { BoxLink } from '~/ui/components/BoxLink';
+import { Collapsible } from '~/ui/components/Collapsible';
+import { Step } from '~/ui/components/Step';
+import { Terminal } from '~/ui/components/Snippet';
+
+The New Architecture is a name that we use to describe a complete refactoring of the internals of React Native, to solve limitations of the original React Native architecture that were discovered over years of usage in production at Meta and other companies.
+
+In this guide, we'll talk about how to use the New Architecture in Expo projects today.
+
+To learn more about the ideas and motivations behind the New Architecture, we recommend the following resources:
+
+<BoxLink
+  title="Introduction to the New Architecture"
+  description="A starting point for exploring the various topics related to the New Architecture."
+  href="https://reactnative.dev/docs/the-new-architecture/landing-page"
+  Icon={BookOpen02Icon}
+/>
+
+<BoxLink
+  title="Why a New Architecture"
+  description="A high level overview of the motivations behind the New Architecture."
+  href="https://reactnative.dev/docs/the-new-architecture/why"
+  Icon={BookOpen02Icon}
+/>
+
+## Why invest in migrating to the New Architecture?
+
+The New Architecture is the future of React Native — yet, for many apps, there may not be many immediate benefits to migrating today. You may want to think of migrating to the New Architecture as an investment in the future of your app, rather than as a way to immediately improve your app.
+
+The changes that come with the New Architecture address technical debt of the original implementation and unlock possibilities for solving long standing issues in React Native, such as interoperability with synchronous native APIs (eg: `UITableView`), and pave the way for [concurrent React support](https://react.dev/blog/2022/03/29/react-v18#what-is-concurrent-react).
+
+## Expo tools and the New Architecture
+
+As of SDK 49, all `expo-*` packages in the [Expo SDK](/versions/latest/) support the New Architecture, with the exception of `expo-updates`. We plan to add support for `expo-updates` in SDK 51.
+
+Additionally, all modules written using the [Expo Modules API](/modules/overview/) support the New Architecture by default! So if you have built your own native modules using this API, no additional work is needed to use them with the New Architecture.
+
+## Initializing a new project with the New Architecture
```

Comment:
```suggestion
## Initialize a new project with the New Architecture
```

Since we are moving to use less gerunds and rely more on short form verbs for section headings, and as this is a new doc, so let's update this to maintain concistency.